scripts/generate-dependencies/generate-dependencies.js

```diff
@@ -1,46 +1,27 @@
 /**
- * Adds package dependencies to the ripple package.json file based on available
- * packages.
+ * Adds package dependencies to:
+ * * Root `./package.json`
+ * * Components `./packages/.../package.json`
  *
- * Requires each ripple package to have:
+ * Requires each ripple component package to have:
  *
- *   1. A `package.json` file with:
- *     - name
+ * 1. A `package.json` file with:
+ *   - name
+ * 2. A `lerna.json` file with:
+ *   - version
  *
  * Script expects to be called via npm in the root directory:
  *
  *   `npm run package-dependencies`
```

Comment:
Hi @alan-cole could you please add this into CONTRIBUTE.md.

Reply:
Added requirements to contribute.md